src/tests/test_dot_notation.py

```diff
@@ -10,13 +10,13 @@ class ChildEntity(Entity):
     slug = fields.StringField()
     pos = PositionEntity
 
-class TestEntity(Entity):
+class ExampleTestEntity(Entity):
```

Comment:
pytest was trying to use this class as a source for tests and warning about it, hence the rename.